web-elements/src/test/java/com/demoqa/ButtonsTest.java

```diff
@@ -0,0 +1,127 @@
+package com.demoqa;
+
+import com.demoqa.page.ButtonsPage;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class ButtonsTest extends BaseTest {
+    @Test
+    public void testPageIsOpenTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
+        String currentURL = driver.getCurrentUrl();
+        String expectedURL = "https://demoqa.com/buttons";
+        assertEquals(currentURL,expectedURL);
+    }
+    @Test
+    public void doubleLeftClickByDoubleClickButtonTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
+        buttonsPage.doubleClickLeft(buttonsPage.doubleClickButton);
+        String expectedResult= "You have done a double click";
+        String actualResult=buttonsPage.getSuccessMessage(buttonsPage.doubleClickMessage);
+        assertEquals(expectedResult, actualResult);
+    }
+    @Test
+    public void oneLeftClickByDoubleClickButtonTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
+        buttonsPage.oneClickLeft(buttonsPage.doubleClickButton);
+        String expectedResult="Element not found";
+        String actualResult = buttonsPage.doubleClickMessageIsAvailable();
+        assertEquals(expectedResult, actualResult);
+    }
+    @Test
+    public void oneRightClickByDoubleClickButtonTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
+        buttonsPage.oneClickRight(buttonsPage.doubleClickButton);
+        String expectedResult="Element not found";
+        String actualResult = buttonsPage.doubleClickMessageIsAvailable();
+        assertEquals(expectedResult, actualResult);
+    }
+    @Test
+    public void doubleRightClickByDoubleClickButtonTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
+        buttonsPage.doubleClickRight(buttonsPage.doubleClickButton);
+        String expectedResult= "Element not found";
+        String actualResult = buttonsPage.doubleClickMessageIsAvailable();
+        assertEquals(expectedResult, actualResult);
+    }
+    @Test
+    public void doubleLeftClickByRightClickButtonTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
+        buttonsPage.doubleClickLeft(buttonsPage.rightClickButton);
+        String expectedResult= "Element not found";
+        String actualResult=buttonsPage.rightClickMessageIsAvailable();
+        assertEquals(expectedResult, actualResult);
+    }
+    @Test
+    public void oneLeftClickByRightClickButtonTest(){
```

Comment:
What do we test in this case?
Why we expected "Elements not found"?
Which manual test case do we automate here?
Do we check "Element not found" message in the manual test?)))